### PR TITLE
commonlib: add solution to Insufficient Process Validation

### PR DESCRIPTION
**Pull request by beraoudabdelkhalek**
## Overview
Added solutions to Insufficient Process Validation vulnerability (vuln_item_wasc_40)

## Related Issues

Related to: [#8056](https://github.com/zaproxy/zaproxy/issues/8056)

## Checklist
- [ ] Update help
- [ ] Update changelog
- [ ] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [ ] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

**Comment by kingthorin** [comment]
Please update the "Unreleased" section of the CHANGELOG.md. There should be other examples pertaining to similar updates.

**Comment by beraoudabdelkhalek** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by beraoudabdelkhalek** [comment]
Is it good now?

**Comment by psiinon** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [bfad9abf\-2133\-4cc4\-a254\-996e9b22e8b5](https://eu.ast.checkmarx.net/projects/0da811fa-6373-42b1-bbb6-d0026285b136/scans?id=bfad9abf-2133-4cc4-a254-996e9b22e8b5&branch=beraoudabdelkhalek%2Fmain)
### No New Or Fixed Issues Found

File: addOns/commonlib/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
+- Add solutions to Insufficient Process Validation vulnerability (Issue 8056).
```
Comment (kingthorin) [inline addOns/commonlib/CHANGELOG.md:9]:
```suggestion
- Add solutions to Insufficient Process Validation vulnerability (Issue 8056).

```

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**